packages/truffle/test/scenarios/commands/build.js

```diff
@@ -9,8 +9,7 @@ describe("truffle build [ @standalone ]", () => {
   let config, project;
 
   describe("when there is no build script in config", () => {
-    beforeEach("set up sandbox", function () {
-      this.timeout(10000);
+    beforeEach("set up sandbox", () => {
```

Comment:
[Mocha says](https://mochajs.org/#arrow-functions) to use `function () {}` not `() => {}`:

```suggestion
    beforeEach("set up sandbox", function () {
```

Reply:
Yeah we talked about this before but I don't think we all agreed on it. We seem to use both styles interchangeably. Going forward do we want to make it convention to do this? I'll bring it up in the next Truffle meeting.